packages/uv/build.sh

```diff
@@ -6,7 +6,7 @@ TERMUX_PKG_MAINTAINER="@termux"
 TERMUX_PKG_VERSION="0.6.9"
```

Comment:
You'll have to add
```suggestion
TERMUX_PKG_VERSION="0.6.9"
TERMUX_PKG_REVISION=1
```
here so it's picked up as a newer version.

Reply:
done